### ISSUE TITLE
Dev-Lite testing for gravity-and-orbits/1.5.0-dev.6, to test phet-io infrastructure changes

### ISSUE THREAD

**Issue by samreid** [phetsims/QA#654]
After publishing Natural Selection, we realized we didn't have a straightforward migration path for clients to go from older sim versions to new versions.  We opened this epic issue to track it: https://github.com/phetsims/phet-io/issues/1763. The sub-issues from that epic issue blocked publication for a while, but have mostly been solved.  There are a few remaining issues to be verified in master, and there were significant structural changes that will need to be tested.  

In discussion on Slack, @samreid proposed testing ph-scale since it had a similar dev test in https://github.com/phetsims/QA/issues/650 or natural-selection, since it was recently tested, but predated these significant changes.  @pixelzoom said it's OK to use either of those, but he cannot be involved right now due to time constraints.  We could also test gravity-and-orbits, which went through RC testing in March.

I decided to open this issue so it is clear that the unblocking parts are ready to go, and that the next step is QA testing.  On slack @KatieWoe said QA has a few things in the queue right now, and there is training happening soon, so it's unclear when testing would start. When QA is ready for testing, we can pick a sim.

**Comment by samreid** [comment]
@kathy-phet said on slack:

> I didn't read through this whole thread as I'm just skimming, but we want to re-publish everything onto this new framework, so @samreid why don't we start with Gravity and Orbits since that is a sim you led and can respond to issues with. If that sounds good, can you check in with @arouinfar on this to make sure there are no other changes needed for that sim and send that one into QA?

@arouinfar can you check whether Gravity and Orbits is ready for another publication?

**Comment by arouinfar** [comment]
@samreid I looked through the GAO repo and there aren't any outstanding design issues to address. There are two deferred dev issues that you should assess before moving forward https://github.com/phetsims/gravity-and-orbits/issues/306 and https://github.com/phetsims/gravity-and-orbits/issues/376

**Comment by samreid** [comment]
@KatieWoe, @kathy-phet, @arouinfar, gravity-and-orbits/1.5.0-dev.5 is ready for light dev testing. Document issues in https://github.com/phetsims/gravity-and-orbits/issues and link to this issue.

This test should focus on PhET-iO, to test our new "upgrading versions" infrastructure.

https://phet-dev.colorado.edu/html/gravity-and-orbits/1.5.0-dev.5/phet/gravity-and-orbits_en_phet.html
https://phet-dev.colorado.edu/html/gravity-and-orbits/1.5.0-dev.5/phet-io/

Please test the following issues, close if they look ok:
https://github.com/phetsims/studio/issues/217
https://github.com/phetsims/phet-io-wrappers/issues/418
https://github.com/phetsims/phet-io/issues/1783

Additionally, please test PhET-iO features, such as studio save/load, playback wrapper, etc.  This is a dev-lite test to identify coarse cross-platform issues.  When this is complete, we will move to an RC test.

Assigning @kathy-phet and @KatieWoe for prioritization.

This test is for (select all that apply) [general, a11y, and phet-io]. Platform coverage is not important for this test. Testing should be around 4 hours max.

- [ ] Tester 1 (Tester = , Platform = , Time = {max 2 hours}
- [ ] Tester 2 (Tester = , Platform = , Time = {max 2 hours}

What to Test
- Click every single button.
- If there is sound, make sure it works.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.
- Try to include browser version numbers
- PhET-iO features

**Comment by samreid** [comment]
I updated the version to dev.6 to include testing for https://github.com/phetsims/chipper/issues/1044

**Comment by liam-mulhall** [comment]
Me on Slack:

> Hey, sorry to bother you so soon after getting back, but is there a link to test https://github.com/phetsims/phet-io-wrappers/issues/418 for the GaO test? I think in order to verify that it's fixed I'd need a link to a GaO with the old API format per https://github.com/phetsims/phet-io-wrappers/issues/418#issuecomment-847304087.

@samreid on Slack:

> From looking at the ph-scale example, it appears the “old format” means the flat list.  I checked gravity and orbits 1.4, and it has the flat structure, so I think it would be a good target for comparison:
https://phet-io.colorado.edu/sims/gravity-and-orbits/1.4/

**Comment by KatieWoe** [comment]
It looks like QA is done

**Comment by samreid** [comment]
Thanks!  I made a placeholder for the upcoming RC in https://github.com/phetsims/QA/issues/657
